test/integration/integration_test.go

```diff
@@ -0,0 +1,194 @@
+// +build integration
+
+package integration
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+	"testing"
+	"time"
+
+	stub "github.com/openshift/cluster-ingress-operator/pkg/stub"
+	"github.com/openshift/cluster-ingress-operator/test/manifests"
+
+	sdk "github.com/operator-framework/operator-sdk/pkg/sdk"
+	k8sutil "github.com/operator-framework/operator-sdk/pkg/util/k8sutil"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+var operatorNamespace string
+var clusterName string
```

Comment:
We may need a better way to ensure operator ns and cluster name is set once we add more tests.